tests/Unit/Elliptic/DiscontinuousGalerkin/Test_DgOperator.cpp

```diff
@@ -0,0 +1,668 @@
+  using FixedSources = typename fixed_sources_tag::type;
+
+  // Get a list of all elements in the domain
+  auto domain = domain_creator.create_domain();
+  const auto initial_ref_levs = domain_creator.initial_refinement_levels();
+  const auto initial_extents = domain_creator.initial_extents();
+  std::unordered_set<ElementId<Dim>> all_element_ids{};
+  for (const auto& block : domain.blocks()) {
+    auto block_element_ids =
+        initial_element_ids(block.id(), initial_ref_levs[block.id()]);
+    for (auto& element_id : block_element_ids) {
+      all_element_ids.insert(std::move(element_id));
+    }
+  }
+
+  ActionTesting::MockRuntimeSystem<Metavars> runner{
+      tuples::TaggedTuple<domain::Tags::Domain<Dim>,
+                          ::elliptic::dg::Tags::PenaltyParameter,
+                          ::Tags::AnalyticSolution<AnalyticSolution>>{
+          std::move(domain), penalty_parameter, analytic_solution}};
+
+  // DataBox shortcuts
+  const auto get_tag = [&runner](
+      auto tag_v, const ElementId<Dim>& local_element_id) -> const auto& {
+    using tag = std::decay_t<decltype(tag_v)>;
+    return ActionTesting::get_databox_tag<element_array, tag>(runner,
+                                                              local_element_id);
+  };
+  const auto set_tag = [&runner](auto tag_v, auto value,
+                                 const ElementId<Dim>& local_element_id) {
+    using tag = std::decay_t<decltype(tag_v)>;
+    ActionTesting::simple_action<element_array,
+                                 ::Actions::SetData<tmpl::list<tag>>>(
+        make_not_null(&runner), local_element_id, std::move(value));
+  };
+
+  // Initialize all elements in the domain
+  for (const auto& element_id : all_element_ids) {
+    ActionTesting::emplace_component_and_initialize<element_array>(
+        &runner, element_id,
+        {initial_ref_levs, initial_extents, FixedSources{}});
+    while (
+        not ActionTesting::get_terminate<element_array>(runner, element_id)) {
+      ActionTesting::next_action<element_array>(make_not_null(&runner),
+                                                element_id);
+    }
+    set_tag(TemporalIdTag{}, 0_st, element_id);
+    const auto& mesh = get_tag(domain::Tags::Mesh<Dim>{}, element_id);
+    const size_t num_points = mesh.number_of_grid_points();
+    const auto& inertial_coords =
+        get_tag(domain::Tags::Coordinates<Dim, Frame::Inertial>{}, element_id);
+    FixedSources fixed_sources{num_points, 0.};
+    fixed_sources.assign_subset(
+        variables_from_tagged_tuple(analytic_solution.variables(
+            inertial_coords, typename fixed_sources_tag::tags_list{})));
+    set_tag(fixed_sources_tag{}, std::move(fixed_sources), element_id);
+  }
+  ActionTesting::set_phase(make_not_null(&runner), Metavars::Phase::Testing);
+
+  if constexpr (Linearized) {
+    INFO(
+        "Test imposing inhomogeneous boundary conditions on source, i.e. b -= "
+        "A(x=0)");
```

Comment:
Are there any checks in this block?  If not, an `INFO` doesn't do anything.

Reply:
The `INFO` also prints when Catch2 fails a test for other reasons, such as uncaught exceptions. I think I put this in because I had a bug here at some point, but I don't remember exactly. I can take it out if you prefer.